### DISCUSSION TITLE
Will spamassassin work without installing amavis and clamav

### DISCUSSION THREAD

**Discussion by rajeevrrs** [modoboa/modoboa#2699] [Installer]
Will spamassassin work without installing amavis and clamav IN Modoboa?

**Comment by tonioo** [comment]
You're free to use spamassassin directly with postfix but that's not something modoboa installer supports.

**Reply by rajeevrrs** [reply]
While Installing Modoboa.

 Can i select false for Amavis and Clamav 
and
True for spamassassin 

Will spamassassin work well?

**Reply by tonioo** [reply]
No, as I said it won't work, it is not supported by the installer
